dev-requirements.txt

```diff
@@ -2,7 +2,6 @@
 flake8==4.0.1
 black==21.12b0
 websockets==10.1
-maturin==0.12.11
```

Comment:
Why did you delete this?

Reply:
Ah, it was simply because installing from the dev-requirements caused an error, since the requirements.txt file already had maturin in it, and so there was a double reference to maturin. I had meant to revert it back before posting the PR and then consult someone about it.

Reply:
@kliwongan , I understand now. Maybe we can just keep `maturin` instead of `maturin==0.12.11` ? That way we won't have conflicts